tests/integration/test_web_client.py

```diff
@@ -92,6 +92,14 @@ def test_creation_lifecycle(self, antares_web: AntaresWebDesktop):
         assert area_ui.x == area_ui.x
         assert area_ui.color_rgb == area_ui.color_rgb
 
+        # tests study reading method and comparing ids, name, areas and settings
+        actual_study = read_study_api(api_config, study.service.study_id)
+
+        assert study.service.study_id == actual_study.service.study_id
+        assert study.name == actual_study.name
+        assert study.version == actual_study.version
+        assert study.get_areas()["fr"].id == actual_study.get_areas()["fr"].id
```

Comment:
I would prefer you check that `list(study.get_areas.keys()) == list(actual_study.get_areas.keys())`.
Also, for the first area of the list, it could be nice to ensure that the areas contain the same renewable and clusters (just check the names)